src/Security/CookiePolicy/test/CookieChunkingTests.cs

```diff
@@ -145,5 +146,48 @@ public void DeleteChunkedCookieWithOptions_AllDeleted()
                 "TestCookieC7=; expires=Thu, 01 Jan 1970 00:00:00 GMT; domain=foo.com; path=/; secure",
             }, cookies);
         }
+
+
+
+        [Fact]
+        public void DeleteChunkedCookieWithOptionsAndResponseCookies_AllDeleted()
+        {
+            var chunkingCookieManager = new ChunkingCookieManager();
+            HttpContext httpContext = new DefaultHttpContext();
+            
+            httpContext.Request.Headers["Cookie"] = new[]
+            {
+                "TestCookie=chunks-7; domain=foo.com; path=/; secure",
```

Comment:
The Cookie header is a single line and doesn't include any of the attributes.
```suggestion
                "TestCookie=chunks-7, TestCookieC1=abcdefghi, ..."
```

Reply:
I changed it to not include the attributes, but I still have have it in multiple lines as the other tests for readability